core/src/HIP/Kokkos_HIP_Space.cpp

```diff
@@ -514,6 +514,46 @@ void* SharedAllocationRecord<Kokkos::Experimental::HIPSpace, void>::
   return r_new->data();
 }
 
+void* SharedAllocationRecord<Kokkos::Experimental::HIPHostPinnedSpace, void>::
+    allocate_tracked(const Kokkos::Experimental::HIPHostPinnedSpace& arg_space,
+                     const std::string& arg_alloc_label,
+                     const size_t arg_alloc_size) {
+  if (!arg_alloc_size) return nullptr;
```

Comment:
Do you know where these overloads are used? Was there a test that would fail otherwise?

Reply:
I believe they were required to compile LAMMPS

Reply:
I believe they were needed for LAMMPS

Reply:
I was asking because it seems that there is no test hitting this code path. We should add one.